### ISSUE TITLE
I don't find CE(cross entropy) in the loss.py

### ISSUE THREAD

**Issue by USTCYYX** [uma-pi1/kge#129]
I want to use this loss function to reproduce the model. Can you show me the code?

**Comment by rufex2001** [comment]
Use KL for KL-divergence, as they are equivalent

**Comment by rufex2001** [comment]
You can see the configs we used here <https://github.com/uma-pi1/kge-iclr20>
.

**Comment by USTCYYX** [comment]
I find ce in the KL, thanks!
